lib/transactions/models.py

```diff
@@ -92,21 +94,23 @@ def is_refunded(self):
             status=constants.STATUS_COMPLETED).exists()
 
     def for_log(self):
-        return (
-            'v5',  # Version.
-            self.uuid,
-            self.created.isoformat(),
-            self.modified.isoformat(),
-            self.amount,
-            self.currency,
-            self.status,
-            self.status_reason,
-            self.buyer.uuid if self.buyer else None,
-            self.seller_product.seller.uuid if self.seller_product else None,
-            self.source,
-            self.carrier,
-            self.region,
-            self.provider)
+        return OrderedDict((
+            ('version', 'v5'),  # Version.
+            ('uuid', self.uuid),
+            ('created', self.created.isoformat()),
+            ('modified', self.modified.isoformat()),
+            ('amount', self.amount),
+            ('currency', self.currency),
+            ('status', self.status),
+            ('reason', self.status_reason),
+            ('buyer', self.buyer.uuid if self.buyer else None),
+            ('seller', (self.seller_product.seller.uuid
+                        if self.seller_product else None)),
+            ('source', self.source),
+            ('carrier', self.carrier),
+            ('region', self.region),
+            ('provider', self.provider)
+        ))
```

Comment:
The `reason` is new here and may break stats. Is there a bug for that? Ideally we'd only append new columns to avoid breaking other things that rely on the order.  See https://github.com/mozilla/monolith-aggregator/blob/master/monolith/aggregator/plugins/solitude.py#L87